docs/guide-ru/rest-controllers.md

```diff
@@ -144,6 +144,10 @@ public function checkAccess($action, $model = null, $params = [])
 {
     // проверить, имеет ли пользователь доступ к $action и $model
     // выбросить ForbiddenHttpException, если доступ следует запретить
+    if ($action === 'update' or $action === 'delete') {
+        if ($model->author_id !== \Yii::$app->user->id)
+            throw new \yii\web\ForbiddenHttpException('You can only '.$action.' articles that you\'ve created.');
```

Comment:
`'You can only '.$action.' articles that you\'ve created.'` → `'sprintf('You can only %s articles that you\'ve created.', $action)`